src/en/readallcomicscom/src/eu/kanade/tachiyomi/extension/en/readallcomicscom/ReadAllComics.kt

```diff
@@ -107,10 +113,15 @@ abstract class ReadAllComics(
         }
     }
 
-    override fun searchMangaRequest(page: Int, query: String, filters: FilterList) =
-        GET("$baseUrl/?story=${query.trim()}&s=&type=${searchType()}", headers)
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = baseUrl.toHttpUrl().newBuilder().apply {
+            addQueryParameter("story", query.trim())
```

Comment:
```suggestion
            addQueryParameter("story", query)
```
Search works without trim